MonitoringSystem/Client/Types/RMSMonitoring.py

```diff
@@ -0,0 +1,40 @@
+"""
```

Comment:
Some more useful comments would be nice, for example what are the various fields

Reply:
Yes, I will add documentation now.